src/routes/api/listingsOrganisations.route.ts

```diff
@@ -0,0 +1,19 @@
+import express from 'express';
+export const router = express.Router();
+import { check } from 'express-validator';
+import { protect, authorise, checkInputError } from '../../middleware';
+import { INVALID_FIELD_MSG } from '../../utils';
+
+// Import listings-organisations controllers
+import { createListingOrganisation, deleteListingOrganisation } from '../../controllers/listingsOrganisations';
+
+// Define input validation
+const validateCreateListingOrganisationEntryFields = [
+    check('listing_id', INVALID_FIELD_MSG('listing id')).isInt(),
```

Comment:
listing_id is a `UUID`, use 
```suggestion
    check('listing_id', INVALID_FIELD_MSG('listing id')).isUUID(),
```